python_base/Dockerfile

```diff
@@ -69,7 +70,8 @@ RUN virtualenv /tmpvenv -p python${INSPIRE_PYTHON_VERSION} && \
     deactivate && \
     mv /tmpvenv/src /src-cache && \
     rm -rf /tmpvenv && \
-    mkdir /code
+    mkdir /code && \
+    dbus-uuidgen > /etc/machine-id
```

Comment:
I think that the issue is more related to using DBus or similar when connecting to the X server. But yep, what you get is that firefox is not running.